apis/hubspot/crm/write/tool.gpt

```diff
@@ -0,0 +1,41 @@
+Name: HubSpot CRM Write
+Type: context
+Description: Read and Write Tools for working with HubSpot CRM
+Share Tools: ./hubspot-crm-write.yaml, ../read
```

Comment:
Rather than sharing the read tool from the other directory, I would recommend sharing the OpenAPI file here.

Reply:
I don't just need the openapi file though, i need the context that is in the tool.gpt and the helpers tools as well.